libretroshare/src/libretroshare.pro

```diff
@@ -327,6 +327,12 @@ INCLUDEPATH *= $${OPENPGPSDK_DIR}
 PRE_TARGETDEPS *= $${OPENPGPSDK_DIR}/lib/libops.a
 LIBS *= $${OPENPGPSDK_DIR}/lib/libops.a -lbz2
 
+################################### Pkg-Config Stuff #############################
+
+LIBS *= $$system(pkg-config --libs $$PKGCONFIG)
```

Comment:
This got duplicated around .pro files which uses libretroshare so you probably need to move it into `use_libretroshare.pri` to avoid duplication

Reply:
Done, but this should be called at end of file because if PKGCONFIG is changed after in .pro file.